multimodal/src/autogluon/multimodal/data/infer_types.py

```diff
@@ -259,6 +260,11 @@ def infer_column_types(
             # No valid index, thus, we will just ignore the column
             column_types[col_name] = NULL
             continue
+        if isinstance(data[col_name][idx], list):
```

Comment:
Do the rois data have more concrete features for data type detection? A list seems a little bit general.

Reply:
To improve code readability, I would suggest:

1. Put the rois detection together with other modalities below.
2. Maybe add the util like `is_rois_column()`. We could add rules inside it in future.

Reply:
Updated the logic to infer rois.